src/core/components/Header/components/Selector/Selector.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, useState } from "react";
+import arrowDown from "@/assets/images/arrow_down.svg";
+import "./Selector.scss";
+
+interface SelectorProps {
+  title: string;
+  setSearchCategory: React.Dispatch<React.SetStateAction<string>>;
+  categories: string[];
+}
+
+const Selector: FC<SelectorProps> = ({
+  title,
+  setSearchCategory,
+  categories,
+}) => {
+  const [showMenu, setShowMenu] = useState<boolean>(false);
+
+  const handleHideMenu = () => {
+    setShowMenu(false);
+  };
+
+  const handleShowMenu = () => {
+    setShowMenu(true);
+  };
+
+  const handleSetCategory = (newCategory: string) => () => {
+    setSearchCategory(newCategory);
+    setShowMenu(false);
```

Comment:
You have function for that

Reply:
Fixed - replace "setShowMenu" with already existed fn "handleHideMenu"